### DISCUSSION TITLE
Good start

### DISCUSSION THREAD

**Discussion by thenexus00** [wpplumber/astroboard#13] [General]
Hey,
Great start.
You  should probably look to do a sweep on the English. "Medias" for example should just be "Media".
You can also replace the main Welcome with custom solutions and I think similar to past packages of this nature having your solution replace the "Welcome" screen is the way to go. Not many will click another tab to see a Dashboard summary.

**Comment by tarikrital** [comment]
Welcome @thenexus00,

I created an issue for your suggestion [here](https://github.com/wpplumber/astroboard/issues/14#issue-2530888069) and the welcome discussion title has been changed.

Thank you for this valuable contribution.
